kybra/compiler/kybra_generate/src/errors/mod.rs

```diff
@@ -143,16 +149,34 @@ impl From<cdk_framework::act::abstract_canister_tree::Error> for crate::Error {
         match value {
             cdk_framework::act::abstract_canister_tree::Error::TypeNotFound(type_ref_name) => {
                 crate::Error::TypeNotFound(format!(
-                    "The type {} is used, but never defined.",
+                    "The type {} is used, but never defined",
```

Comment:
Augment this to tell the user to make sure they did Record, Variant, etc correctly